docs/content/deploy.md

```diff
@@ -28,98 +24,58 @@ To start a local deployment, either:
 
 This starts a local version of the SpatialOS Runtime on your development machine together with all server-workers specified in the launch configuration you used.
 
+> **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
+
 To start your client-workers, from a terminal window, in any directory, run `spatial local worker launch <YourExternalWorker> <YourLaunchConfig>`, replacing the `<example-content>` with names relevant to your game. Use the [Inspector](https://docs.improbable.io/reference/latest/shared/operate/inspector) to look at the current state of your game world.
 
 ## Cloud deployment
 
+To start a cloud deployment follow the section.
+
+### Setup the SpatialOS project name
+
 You are allocated an empty SpatialOS project in the cloud when you sign up to SpatialOS; you use this to deploy your game, but to do this you need to tell the GDK the name of your allocated SpatialOS project so it knows where to deploy your game to.
 
-To start a cloud deployment:
-1.  Open the `spatialos.json` file in the root folder of your game.
+1. Open the `spatialos.json` file in the root folder of your game.
 1. Change the `name` field so it matches the name of your SpatialOS project. You can find this in the SpatialOS [Console](https://console.improbable.io). It’ll be something like `beta_someword_anotherword_000`.
-1. Build and upload a game assembly for the deployment
-    - The assembly includes executable files for the client-workers and server-workers, and the assets both types of workers use (such as the models and textures used by the client-server - that is, the game executable code - to visualise the game).
-    1. To build an assembly; in the Unity Editor, select **SpatialOS** > **Build all workers for cloud**.
 
-    1. To upload an assembly; open a terminal and navigate to the directory in which your game is in (the repository you’ve cloned). Run `spatial cloud upload <assembly name>`.
-    - The `<assembly name>` is a label you create so you can identify this assembly in the next step - for example you could call it `MyGDKAssembly`.
-    > **It’s finished uploading when:** You see a successful upload report printed in your terminal output, and your browser will open a console.improbable.io page dispaying the deployment.
+### Build and upload the game assembly
+The assembly includes executable files for the client-workers and server-workers, and the assets both types of workers use (such as the models and textures used by the client-server - that is, the game executable code - to visualise the game).
```

Comment:
Break this up into multiple sentences, rather than running-on and parenthesizing.

Reply:
-> 
The assembly includes executable files for the client-workers and server-workers, as well as the assets both types of workers use. (The assets might be the models and textures that the client-servers - that is, the game executable code - use to visualise the game.)